src/app/core/filter/list-filter/list-filter.component.html

```diff
@@ -1,30 +1,17 @@
 <mat-form-field appearance="fill" class="full-width">
   <mat-label>{{ filterConfig.label || filterConfig.name }}</mat-label>
-
-  <mat-select
-    [id]="filterConfig.name"
-    [(value)]="filterConfig.selectedOptionValues"
-    (valueChange)="filterConfig.selectedOptionChange.emit($event)"
-    multiple
+  <app-basic-autocomplete
+    [multi]="true"
+    [options]="filterConfig.options"
+    [formControl]="autocompleteControl"
+    [optionToString]="getOptionLabel"
+    [valueMapper]="getOptionValue"
   >
-    <!-- SELECT ALL / CLEAR -->
-    <div class="flex-row">
-      <button mat-flat-button i18n class="util-button" (click)="selectAll()">
-        Select All
-      </button>
-      <button mat-flat-button i18n class="util-button" (click)="deselectAll()">
-        Clear
-      </button>
-    </div>
-
-    @for (option of filterConfig.options; track option.key) {
-      <mat-option
-        [value]="option.key"
-        [appBorderHighlight]="option.color"
-        style="padding-top: 0.5em; padding-bottom: 0.5em"
-      >
-        {{ option.label }}
-      </mat-option>
-    }
-  </mat-select>
+    <ng-container autocompleteHeader>
+      <div class="flex-row">
+        <button mat-flat-button i18n (click)="selectAll()">Select All</button>
```

Comment:
the `class="util-button"` did have a purpose. Without it, the hover background color looks a bit off right now